lldb/test/API/lang/swift/framework_paths/TestSwiftFrameworkPaths.py

```diff
@@ -20,13 +16,25 @@ def test_system_framework(self):
             self, 'break here', lldb.SBFileSpec('main.swift'))
 
         log = self.getBuildArtifact("types.log")
-        self.runCmd('log enable lldb types -f "%s"' % log)
+        self.expect('log enable lldb types -f "%s"' % log)
```

Comment:
Why did you change this to `expect`?

Reply:
I thought that self.runCmd doesn't fail if the command fails. But I may be wrong about this.